### DISCUSSION TITLE
（头脑风暴）如果文管支持将文件或文件夹，在当前目录 置顶 显示 ，如何

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#3976] [Features Request & Ideas | 特性请求 & 头脑风暴]
如题，设为 置顶 的文件或目录，不受排序方式的影响

欢迎大家讨论

**Comment by BLumia** [comment]
cc @meiyixiang 

我个人倒是没这个需求 = =

**Reply by linuxlionking** [reply]
哈哈，我的想法可能过于特殊，情况是这样的，文管的 快捷访问 和 标记，其实也能满足我的需求 ，只是对一些经常访问的目录，有几个目录或文件，是特别高频使用的，想着如果我进到这个主目录，直接置顶显示那几个文件，会更方便

**Comment by meiyixiang** [comment]
我不确定会不会做，因为还没有去深入的思考这个问题需要和文管的产品去综合讨论一下。但是站在我个人的角度我觉得这个功能是个很小的功能，做了可能会提升一点点个性化使用效率，但类似功能做的过多也可能让整个项目太过复杂。

**Reply by linuxlionking** [reply]
纯属抛砖引用，头脑风暴
我也认为，文管应以功能稳定为主
不知将来是否可以支持 插件，开放接口，让社区来自行添加一些个性化小功能
